lib/consumer.ex

```diff
@@ -70,7 +76,10 @@ defmodule GenRMQ.Consumer do
       concurrency: true,
       queue_ttl: 5000,
       retry_delay_function: fn attempt -> :timer.sleep(1000 * attempt) end,
-      reconnect: true
+      reconnect: true,
```

Comment:
how about this structure instead:
```
[
  reconnect: true,
  deadletter: [
    queue: "...",
    exchange: "..."
  ]
]
```
Then `config[:deadletter] == nil` means no deadletter setup.

Reply:
@vorce this would make sense, although not sure how to make it backwards compatible 😕

Reply:
Can always make a major version bump and add some instructions on how to upgrade to the README hehe. But probably not worth this time :)